components/DsmAirdrop/CheckAirdrop.tsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react'
+import { Box, Typography, useTheme, Button, TextField, CircularProgress } from '@material-ui/core'
+import axios from 'axios'
+import useTranslation from 'next-translate/useTranslation'
+import TickIcon from '../../assets/images/icons/icon_tick.svg'
+import ParachuteIcon from '../../assets/images/parachute.svg'
+import SelectAccountDialog from '../SelectAccountDialog'
+import useStyles from './styles'
+
+interface CheckAirdropProps {
+  onConfirm(): void
+  setSelectedAddress: (address: string) => void
+}
+
+const CheckAirdrop: React.FC<CheckAirdropProps> = ({ onConfirm, setSelectedAddress }) => {
+  const classes = useStyles()
+  const { t } = useTranslation('common')
+  const theme = useTheme()
+  const [loading, setLoading] = React.useState(false)
+  const [address, setAddress] = React.useState('')
+  const [verifyData, setVerifyData] = React.useState(null)
+  const [dataStakingInfo, setDataStakingInfo] = React.useState(null)
+  const [lpInfos, setLpInfos] = React.useState(null)
+  const [error, setError] = React.useState(false)
+  const [isSelectAccountDialogOpen, setIsSelectAccountDialogOpen] = React.useState(false)
+
+  const verify = React.useCallback(async () => {
+    try {
+      setLoading(true)
+      const result = await axios.get(`https://api.airdrop.desmos.network/users/${address}`)
```

Comment:
replace axios by fetch so we can have one less dependency
use env variable for api url

Reply:
@calvinkei I think Axios should be acceptable? I have also implemented it in my code
I do agree env variable should be added.